src/main/java/es/thalesalv/chatrpg/adapters/data/entity/PersonaEntity.java

```diff
@@ -51,6 +51,9 @@ public class PersonaEntity {
     @Column(name = "visibility", nullable = false)
     private String visibility;
 
+    @Column(name = "isMultiplayer", nullable = false, columnDefinition = "boolean default false")
+    private boolean isMultiplayer;
```

Comment:
Originally I made it part of the chconf, but then changed to persona because that's where the intent is. This is intrinsically tied to intent, as it's applied to RPG only